xwiki-rendering-api/src/main/java/org/xwiki/rendering/internal/parser/reference/GenericLinkReferenceParser.java

```diff
@@ -102,7 +102,7 @@
     /**
      * Replacements chars for the escapes to be removed from the interwiki content.
      */
-    private static final String[] ESCAPE_REPLACEMENTS_INTERWIKI = new String[] { "" + ESCAPE_CHAR, "" };
+    private static final String[] ESCAPE_REPLACEMENTS_INTERWIKI = new String[] { String.valueOf(ESCAPE_CHAR), " " };
```

Comment:
It's supposed to be an empty string, you modified it to a String containing a white space.